client/state/ui/editor/post/actions.js

```diff
@@ -0,0 +1,71 @@
+/**
```

Comment:
Do we want to discuss that here or in https://github.com/Automattic/wp-calypso/issues/2994?